ibis-server/tests/routers/v2/connector/test_athena.py

```
@@ -0,0 +1,402 @@
+        "1996-01-02 00:00:00.000000",
+        "1_36901",
+        "2024-01-01 23:59:59.000000",
+        "2024-01-01 23:59:59.000000",
+        None,
+        "616263",
+    ]
+    assert result["dtypes"] == {
+        "orderkey": "int64",
+        "custkey": "int64",
+        "orderstatus": "object",
+        "totalprice": "object",  ### fixme this should be float64
```

Pylint (3.3.7)

[warning] ## fixme this should be float64

(W0511)